app/app.go

```diff
@@ -7,12 +7,19 @@ import (
 	"strings"
 )
 
-// extraArgs contains extra arguments to append to
-// os.Args. The arguments are separated with |.
-// Useful for running programs on mobiles where the
-// command line is not available.
-// Set with the go linker flag -X.
-var extraArgs string
+var (
+	// extraArgs contains extra arguments to append to
+	// os.Args. The arguments are separated with |.
+	// Useful for running programs on mobiles where the
+	// command line is not available.
+	// Set with the go linker flag -X.
+	extraArgs string
+	// appID specifies the package name for Android and Wayland
+	// or the bundle id for iOS and tvOS.
+	// appID can be set by using either AppID configuration option
+	// or using gio-cmd -appid flag.
+	appID = "org.gioui.app"
```

Comment:
I'm not sure whether it's a good idea to add the identifier by default, as I understand it should be unique per app.

Reply:
No it's not required to be unique and after all you are allowed to change app_id on runtime. Having default would at least group all Gio application windows together. by default instead showing bunch of "Unknown" windows.

Reply:
Just found that for X11 there is something like that

https://github.com/gioui/gio/blob/dee53b364560b2339551a9082c4b4a8ddcbcb17d/app/os_x11.go#L796-L798

these values should be same, since they serve kind of same purpose

Reply:
That's what I meant, grouping different gio projects together seems weird. Grouping by executable name seems like a better choice for a default behaviour.

Reply:
Ok changed default to be same as for X11